lib/assesment_1/fi_uzair.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+class FIAhmad extends StatelessWidget {
```

Comment:
Please correct the name of the widget, widgets are classes the name should be similar to file name, change it to F1Uzair.

After changing the name please add it to main.dart